manifests/plugin/pulp.pp

```diff
@@ -64,10 +64,10 @@
     enabled       => $pulpnode_enabled,
     listen_on     => $listen_on,
   }
-  -> foreman_proxy::settings_file { 'pulp3':
```

Comment:
Please keep this with `ensure => absent` to keep systems clean. Probably good to add a note that smart_proxy_pulp x.y drops the file and this cleans it up.

Reply:
Good catch. Let me know if this latest version suffices. (Not 100% about the chaining)

Reply:
Technically the whole chaining is too much. Every settings file there requires `Foreman_proxy::Plugin[pulp]' but they don't depend on each other. It'll still work of course.